apps/moose-console/src/app/infrastructure/databases/[databaseName]/tables/[tableId]/page.tsx

```diff
@@ -64,19 +65,19 @@ export default async function Page({
 
   return (
     <section className="p-4 max-h-screen flex-grow overflow-y-auto flex flex-col">
-      <div className="py-10">
-        <div className="text-6xl">
-          <Link className="text-muted-foreground" href={"/"}>
-            ../
+      <div className="text-base text-muted-foreground flex">
+        <Fragment>
```

Comment:
Any reason you're using fragment here vs just <></>? Mainly a curiosity thing

Reply:
Honestly totally forgot it was a thing. Think we used to have a linting rule about it? So just habit